appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -57,6 +57,14 @@
   @Description("Text on \"Overwrite\" button")
   String overwriteButton();
 
```

Comment:
I think it would be great if you add this kind of comment here...

``` java
//Used in editor/youngandroid/properties/YoungAndroidBackgroundImageRepeatModePropertyEditor.java
```
source code has this kind of comments in `OdeMessages.java` file.

Reply:
Sure thanks for suggestion